Agenda/Controllers/OnboardingViewController.swift

```diff
@@ -0,0 +1,168 @@
+    
+    private lazy var scrollView: UIScrollView = {
+        let scrollView = UIScrollView()
+        scrollView.showsVerticalScrollIndicator = false
+        scrollView.translatesAutoresizingMaskIntoConstraints = false
+        return scrollView
+    }()
+    private lazy var contentView: UIView = {
+        let view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+    
+    private lazy var welcomeLabel: UILabel = {
+        let label = UILabel()
+        label.textAlignment = .center
+        label.text = Labels.Onboarding.welcomeLabel
+        label.numberOfLines = 0
+        label.font = UIFont.systemFont(ofSize: 36, weight: .heavy)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    private lazy var tableView: UITableView = {
+        let tableView = UITableView(frame: .zero, style: .insetGrouped)
+        tableView.dataSource = self
+        tableView.sectionHeaderHeight = 0
+        tableView.backgroundColor = .clear
+        tableView.register(OnboardingTableViewCell.self, forCellReuseIdentifier: OnboardingTableViewCell.identifier)
+        tableView.allowsSelection = false
+        tableView.isScrollEnabled = false
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        return tableView
+    }()
+    
+    private lazy var backgroundButtonView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .systemBackground
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+    private lazy var continueButton: UIButton = {
+        let button = UIButton()
+        button.setTitle(Labels.Onboarding.continueButtonLabel, for: .normal)
+        button.titleLabel?.font = UIFont.systemFont(ofSize: 18, weight: .semibold)
+        button.layer.zPosition = 1
+        button.addTarget(self, action: #selector(continueButtonTapped), for: .touchUpInside)
+        button.backgroundColor = .systemRed
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.layer.cornerRadius = 10
+        return button
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        view.backgroundColor = .systemBackground
+        
+        setupView()
+        setContraints()
+    }
+}
+
+// MARK: - Methods
+private extension OnboardingViewController {
+    
+    func setupView() {
+        view.addSubview(scrollView)
+        
+        scrollView.addSubview(contentView)
+        
+        contentView.addSubview(welcomeLabel)
+        if let locale = Locale.current.languageCode, locale == "en" {
+            // Like Apple do
+            let labelText = NSMutableAttributedString(string: welcomeLabel.text ?? "")
+            labelText.addAttribute(NSAttributedString.Key.foregroundColor, value: UIColor.systemRed, range: NSRange(location: 15, length: 6))
+            welcomeLabel.attributedText = labelText
+        }
```

Comment:
Replace hard-coding on english locale to "if you've found Agenda, then color it with `.systemRed`"